### DISCUSSION TITLE
无法接收授邮件

### DISCUSSION THREAD

**Discussion by xzhzzzzzz** [getlantern/lantern-forum-cn#965] [General]
系统重装，想通过注册邮箱地址升级到专业版，QQ邮箱收不到邮件，在垃圾邮件和广告邮件里面都找不到， 之前都可以收到。求助

**Comment by MaxMKMax** [comment]
@xzhzzzzzz 您好. 请您提供我账号邮箱和支付单据. 谢谢您. 我来帮您提交问题给技术团队解决您遇到的问题.
